content/references/protocol-reference/ledger-data/ledger-object-types/accountroot.md

```diff
@@ -9,7 +9,7 @@ labels:
 # AccountRoot
 [[Source]](https://github.com/ripple/rippled/blob/5d2d88209f1732a0f8d592012094e345cbe3e675/src/ripple/protocol/impl/LedgerFormats.cpp#L27 "Source")
 
-The `AccountRoot` object type describes a single [account](accounts.html), its settings, and XRP balance.
+An `AccountRoot` ledger entry type describes a single [account](accounts.html), its settings, and XRP balance.
```

Comment:
Part of an ongoing effort to change from "ledger object" (ambiguous: whole ledger? one item in it?) to "ledger entry" for objects inside a ledger. The recommended terms in our contributing guide have been updated to prefer this usage, but a lot of existing pages haven't been yet.